web/src/games/bingo/utils.ts

```diff
@@ -17,3 +17,22 @@ export function inferActivePlayers(G: IGameState, playerID: string) {
   const shoutCallPlayers = Object.keys(G.players).filter((p) => G.players[p].shoutCount > 0 && p !== playerID);
   return [playerID, ...shoutCallPlayers].slice(0, MAX_BACKOFF_CANDIDATES);
 }
+
+export function gameLocalStore(action: string, matchCode: string, key: string, value: any = null) {
+  if (typeof window !== 'undefined') {
+    const storeKey = 'bingoLocalStore';
+    let storeValue = JSON.parse(localStorage.getItem(storeKey)) || {};
+    if (storeValue.matchCode !== matchCode) {
```

Comment:
Use session storage instead:
https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage

This will work better even if the user has two tabs open.

Reply:
and you wont need to check for the matchCode at all

Reply:
Changed to session storage, but will still need match code. 
Otherwise if you start a new game in same window, you will see previous selection.